src/pages/page404/text404/text404.css

```diff
@@ -0,0 +1,4 @@
+.text {
```

Comment:
such a common selector could be a problem because it will affect all ".text" at any DOM levels